eRecruiter.Api/Responses/Application/ApplicationReferrerResponse.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+
+namespace eRecruiter.Api.Responses
+{
+    /// <summary>
+    /// Response for application queries on referrer.
+    /// </summary>
+    public class ApplicationReferrerResponse
```

Comment:
Would it make sense to add the ``JobId`` to the response, since we can't query an application by it's ID?